src/networkData.ts

```diff
@@ -70,6 +70,17 @@ const versi: NetworkData = {
   rpcEndpoint: "wss://versi-rpc-node-0.parity-versi.parity.io/",
 };
 
+const trappist: NetworkData = {
+  balanceCap: 100,
+  chains: [],
+  currency: "HOP",
+  decimals: 12,
```

Comment:
do we need to hardcode a currency and decimals? I guess they could be loaded dynamically from the onchain config

Reply:
From what I see, [decimals is being used](https://github.com/search?q=repo%3Aparitytech%2Fpolkadot-testnet-faucet%20decimals&type=code) to convert values into human readable or do some big int calculations.

How would you load the amount of decimals from on-chain? 
Taking into consideration that they never change, keeping them hardcoded seems like a practical solution to me

Reply:
I don't feel that loading them dynamically would be an improvement. It would make code considerably more complex, while not giving much in return.  

We still would need these static configs, at least for `explorer`, `rpcEndpoint`, `balanceCap` and `dripAmount` properties. Cutting them in half, while adding all this dynamic initialisation doesn't look like a win to me.